scripts/setup-staging.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+# note call from repo root
```

Comment:
You can make it work relative to the script by using `${BASH_SOURCE%/*}` http://mywiki.wooledge.org/BashFAQ/028